src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/CustomFilter.cs

```diff
@@ -0,0 +1,148 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.MachineLearning.Inference
+{
+    /// <summary>
+    /// The CustomFilter class provides interfaces to create a custom-filter in the pipeline.
+    /// </summary>
+    /// <remarks>
+    /// Because of data translation (i.e. marshaling and unmarshaling ) between Native and .Net Layer, CustomFilter class shows lower performance than native implementation.
+    /// </remarks>
+    /// <since_tizen> 8 </since_tizen>
+    public class CustomFilter : IDisposable
+    {
+        private Interop.Pipeline.CustomEasyInvokeCallback _callback;
+        private Callback _Callback;
```

Comment:
`_callback` and `_Callback` are not good for code readability

I recommend to rename
``` c#
private Interop.Pipeline.CustomEasyInvokeCallback _nativeCallback;
private Callback _filter;
```